### PR TITLE
WT-11405 Improve gdb debug script to dump larger trees

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

File: tools/gdb/wt_debug_script_update.py [hunk 1]
```diff
@@ -148,6 +148,29 @@ def dump_disk(leaf_page):
...
+    dbg('leaf', leaf_page)
+    dump_disk(leaf_page)
+    dump_modified(leaf_page)
+
+def dump_int_page(int_page):
+    dbg('Internal page', int_page)
+    pindex = int_page['u']['intl']['__index'].dereference()
+    dbg('pindex', pindex)
+    num_entries = int(pindex['entries'])
+    #dbg('rp-pre-index', rpindex['index'].dereference().dereference())
```
Comment (clarissecheah) [inline tools/gdb/wt_debug_script_update.py:161]: Is this commented line meant to be added?

File: tools/gdb/wt_debug_script_update.py [hunk 2]
```diff
@@ -148,6 +148,29 @@ def dump_disk(leaf_page):
...
+    for i in range(0, num_entries):
+        if not pindex['index'][i].dereference()['page']:
+            continue
+        page = pindex['index'][i].dereference()['page'].dereference()
+        dbg('page', page)
+        page_type = page['type']
+        dbg('page type', page_type)
+        if page_type == 3 or page_type == 6:
```
Comment (clarissecheah) [inline tools/gdb/wt_debug_script_update.py:169]: Could we add a short comment here to describe the page names corresponding with types 3 and 6?

Comment (kommiharibabu) [reply tools/gdb/wt_debug_script_update.py:170]: Added a comment. Please check and let me know.

**Review by clarissecheah — APPROVED** [review]
LGTM, thanks! :)

**Review by agorrod — APPROVED** [review]
Thanks @kommiharibabu lgtm

**Comment by kommiharibabu** [comment]
evergreen merge

**Pull request merged**